libsolidity/analysis/TypeChecker.cpp

```diff
@@ -199,7 +196,7 @@ void TypeChecker::checkContractAbstractFunctions(ContractDefinition const& _cont
 				if (!function->isImplemented())
 					// Base contract's constructor is not fully implemented, no way to get
 					// out of this.
-					allBaseConstructorsImplemented = false;
+					_contract.annotation().unimplementedFunctions.push_back(function);
```

Comment:
`annotation().allBaseConstructorsImplemented`, but also include it in the `unimplementedFunctions` list? ?